go.mod

```diff
@@ -8,6 +8,7 @@ require (
 	github.com/iancoleman/strcase v0.2.0
 	github.com/microsoft/kiota-authentication-azure-go v0.4.1
 	github.com/microsoft/kiota-serialization-json-go v0.5.6
+	github.com/microsoftgraph/msgraph-beta-sdk-go v0.39.0
```

Comment:
Since this uses another SDK (beta), do you know how many controls in the Microsoft 365 mod use this table? The main point I'm considering is it worth it to increase build times for this table, based on how generally useful it is and how many controls rely on it

Reply:
Also, does the GA version of the SDK support this yet, or only beta still?

Reply:
@cbruno10 I checked for the API in GA and found one that gives the same data. I have updated the table code to use that API.